bundles/org.openhab.binding.ecotouch/README.md

```diff
@@ -0,0 +1,68 @@
+# EcoTouch Binding
+
+The openHAB EcoTouch binding allows interaction with a [Waterkotte](https://www.waterkotte.de/) heat pump.
+
+## Supported Things
+
+| Thing type      | Description                                      |
+|-----------------|--------------------------------------------------|
+| geo             | Waterkotte EcoTouch Geo + EcoVent                |
+| air             | Waterkotte EcoTouch Air + EcoVent                |
+
+This binding was tested with a Waterkotte DS 5027 Ai DS 5010.5Ai geothermal heat pump).
+
+## Discovery
+
+Discovery is not supported. You need to provide the IP address of the display unit of the heatpump.
+
+## Binding Configuration
+
+No Binding configuration required.
+
+## Thing Configuration
+
+Things can be fully configured via the UI. The following information is useful when configuring things via thing configuration files.
+
+```
+Thing ecotouch:geo:9a0ec0bbbd "Waterkotte Heatpump" @ "basement" [ ip="192.168.1.100", username="admin", password="wtkadmin", refresh=120 ]
+```
+
+| Property                        | Type    | Default | Required | Description |
+|---------------------------------|---------|---------|----------|-------------|
+| ip                              | String  |         | yes      | IP address or hostname of the display unit of the heat pump |
+| username                        | String  |         | yes      | since software version 1.6.xx of the display unit: "waterkotte"; previously "admin". |
+| password                        | String  |         | yes      | since software version 1.6.xx of the display unit: "waterkotte"; previously "wtkadmin". |
+| refresh                         | Integer |      60 | no       | time in s after which all channels will be requested again from the heat pump |
+
+## Channels
+
+All available channels can be seen inside PaperUI. Here is a small extract:
+
+| Channel ID          | Type               | Read-Only | Description |
+|---------------------|--------------------|-----------|-------------|
+| temperature_outside | Number:Temperature | yes       | The current outside temperature |
```

Comment:
```suggestion
| temperatureOutside | Number:Temperature | yes       | The current outside temperature |
```
Please change all channels to use lowerCamelCase

see

https://www.openhab.org/docs/developer/guidelines.html#java-coding-style

Reply:
First, thanks for reviewing! Yes this is a OH1 binding upgrade and all channels have been kept compatible to OH1 to minimize the migration work for the users. What should I do now?

Reply:
Leave it as is until someone with more knowledge steps in to make the call on what to do. I’ll help with what I can.

Reply:
As far as I know underscores are still allowed in OH3.